src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/SetAttributeCommand.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.References
+{
+    internal class SetAttributeCommand : IProjectSystemUpdateReferenceOperation
```

Comment:
>SetAttributeCommand [](http://example.com/codeflow?start=19&length=19)

SetAttributeCommand (and the unset variant) seem very generic to me given that it actually does. What do you think about a more specific name like (Un)SetTreatAsUsedAttributeCommand?

Reply:
done